qemu/tests/cpu_device_hotplug.py

```diff
@@ -0,0 +1,109 @@
+import logging
+import re
+import time
+
+from autotest.client.shared import error
+from autotest.client.shared import utils
+
+from virttest import utils_misc
+
+
+@error.context_aware
+def run(test, params, env):
+    """
+    Runs vCPU hotplug tests based on CPU device:
+    """
+    def get_cpu_model():
```

Comment:
Oh, got it. But I think that it better to split it to your incoming commit. Because `grep` all your code not found where it used and you cpu module pass via test params dict now,  that make reader in trouble.

Thanks,
Xu